scheds/rust/scx_bpfland/src/bpf/main.bpf.c

```diff
@@ -234,7 +234,7 @@ static int calloc_cpumask(struct bpf_cpumask **p_cpumask)
 	struct bpf_cpumask *cpumask;
 
 	cpumask = bpf_cpumask_create();
-	if (!cpumask)
+	if (unlikely(!cpumask))
```

Comment:
Is there any actual benefit using likely/unlikely in BPF code? I think the JIT will just ignore this hint. Let's drop this for now.